edgexfoundry/app-functions-sdk-go
# refactor!: Replace internal topics from config with new constants

BREAKING CHANGE: Internal topics no longer configurable, except the base topic. Trigger topics for edgex-messagebus and external-mqtt now directly under Trigger section.

Signed-off-by: Leonard Goodell <leonard.goodell@intel.com>







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **Existing tests suffice**
      TBD

## Testing Instructions
Run non-secure EdgeX stack 
Build template app from this branch
Run template app locally
Verify it receives events and publish data as expected. Log message should contains:
```
level=DEBUG ts=2023-02-13T22:34:54.9603438Z app=app-new-service source=runtime.go:422 msg="Event Received with ProfileName=Random-Integer-Device, DeviceName=Random-Integer-Device and ReadingCount=1"
level=INFO ts=2023-02-13T22:39:33.6275216Z app=app-new-service source=messaging.go:94 msg="Publishing to topic: 'event-xml'"

```

## New Dependency Instructions (If applicable)
